draft-ietf-rats-eat.md

```diff
@@ -1287,6 +1287,31 @@ A profile document doesn't have to be in any particular format. It may be simple
 
 A profile may define, and possibly register, one or more new claims if needed. A profile may also reuse one or more already defined claims, either as-is or with values constrained to a subset or subrange.
 
+
+## Full and Partial Profiles
+
+A "full" profile is one that fully guarantees interoperability when a sender and receiver both adhere to it.
```

Comment:
Should a profile include trust anchor key material or a reference to key material? This is required to "fully guarantee  interoperability". At present, only a key ID is included.

Reply:
#417 

It seems like a slippery slope to start discussing key formats, key distribution and key trust.

Reply:
There's no need to get into discussing key formats, etc. Maybe something like:

```
A "full" profile is one that fully guarantees interoperability when a sender and receiver both adhere to it and have access to any necessary verification, decryption or MAC key(s), as discussed in sections 6.2.9 and 9.6.
```